source/install/cloud-changelog.md

```diff
@@ -4,12 +4,54 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
 
 Latest Mattermost Cloud releases:
 
+- [Release 2023-09-28](#release-2023-09-28)
 - [Release 2023-09-12](#release-2023-09-12)
 - [Release 2023-08-29](#release-2023-08-29)
 - [Release 2023-08-09](#release-2023-08-09)
 - [Release 2023-08-03](#release-2023-08-03)
 - [Release 2023-07-26](#release-2023-07-26)
-- [Release 2023-07-20](#release-2023-07-20)
+
+## Release 2023-09-28
+
+### Improvements
+
+#### User Interface (UI)
+ - GitLab plugin v1.7.0 is now available via prepackaged plugins.
```

Comment:
Suggestion: Document all prepackaged plugins consistently. Consider the Calls bullet point below as our starting point: ``Pre-packaged Calls version v0.19.1.``

In this case: ``Pre-packaged GitLab plugin version v1.7.0.``